files-community/Files
# Feature: Added support for formatting drives from the properties window

**Resolved / Related Issues**
   Closes #11607
   Related #13142 

**Details**
As pointed out in #11607, `Format...` option is already visible in the main context menu of drives.
I added a `Format` button to the properties window of drives.
After #13142, `Turn on BitLocker` was visible in the main menu only when the drive required admin privileges; this PR fixes that bug

**Validation**
How did you test these changes?
   1. Open properties of a drive
   2. Click on `Format...`
   3. See that the format dialog pops up
   4. Right click on a USB stick and see that `Turn on BitLocker` is in the main menu

**Screenshots (optional)**
![Screenshot 2023-08-10 110904](https://github.com/files-community/Files/assets/102259289/98c26c49-7165-4c0e-b069-e2d8eb3ef003)
